server/main.js

```diff
@@ -1,7 +1,8 @@
 import { Meteor } from 'meteor/meteor';
 
+import '../imports/api/Admin';
 import { Players } from '../imports/api/Players.js';
-import { HongKongHands } from '../imports/api/GameDatabases.js'
+import '../imports/api/GameDatabases.js'
```

Comment:
Are these additions even necessary?  I see no need for admin or game database stuff here

Reply:
You need to import it otherwise it's unused.

Reply:
This was an issue we had with not saving HK hands.